agrirouter-api-java-api/src/main/kotlin/com/dke/data/agrirouter/api/service/parameters/MessageConfirmationForAllPendingMessagesParameters.kt

```diff
@@ -15,12 +15,5 @@ class MessageConfirmationForAllPendingMessagesParameters : ParameterValidation {
     @NotNull
     lateinit var onboardingResponse: OnboardingResponse
 
-    @NotNull
```

Comment:
Are those changes part of the current PR? Mixing different functional features within one PR is a PITA.

(ง'̀-'́)ง

Reply:
Yep, that should be part of a different pull request indeed. Good eyes :cop: